CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Current Develop Branch
 
+## 4.12.0 Tuesday September 27 2018
```

Comment:
Today is Thursday 😄

Reply:
Apparently my `version:bump` script doesn't help me out when there's no changelog yet, so I copy-pasted and now pay for it.